doc/source/building/working_with_images/vagrant_setup.rst

```diff
@@ -69,21 +70,40 @@ the following changes are required:
       </users>
 
    This adds the **vagrant** user to the system and applies the
-   name of the user as the password for login. Change this as you
-   see fit.
+   name of the user as the password for login.
 
 4. Integrate public SSH key
 
-   Reach out to
-   `Insecure Keys <https://github.com/hashicorp/vagrant/tree/master/keys>`_
-   for details on this keys. Add the public key to the
-   box as overlay file in your image description at
-   :file:`home/vagrant/.ssh/authorized_keys`
+   Vagrant requires an insecure public key pair [#f1]_ to be added to the
```

Comment:
No space between the word and the footnote. So it should be "key pair¹" instead of "key pair ¹".

Reply:
The space is mandatory in sphinx: the footnote will not be resolved when the space is missing.

Reply:
You're absolutely right. Sorry, ignore my idea.